README.md

```diff
@@ -14,6 +29,15 @@ query string. If the call is successful the generated content of the specificati
 * `url` (required). The URL of the draft to fetch and generate from.
 * `publishDate`. The date at which the publication of this draft is supposed to occur.
 
+### As a Node.js module
+
+```js
+const SPEC_GEN = require('w3c-spec-generator');
```

Comment:
Making it upper case like that is not very js? why not just `const specGen = `

Reply:
I admit it's not common&hellip; but [I'm not the only one writing their constants in uppercase, or some of them](https://github.com/francoischalifour/medium-zoom/blob/master/src/medium-zoom.js)! :)

Shall I change them all to usual camelCase?